packages/js-moi-providers/__tests__/jsonrpc-provider.test.ts

```diff
@@ -24,10 +27,10 @@ describe("Test JsonRpcProvider Query Calls", () => {
         fuel_limit: 200,
         payload: {
             standard: AssetStandard.MAS0,
-            symbol: "TESTING",
-            supply: 1248577
+            symbol: "TEST #" + Math.floor(Math.random() * 1000),
```

Comment:
nitpick, can we have utility function to generate random symbol?

Reply:
4535277a344fe6e3a959a6a82168025d98f45afa